config/locales/en.yml

```diff
@@ -633,6 +633,8 @@ en:
           attributes:
             application:
               <<: *application
+            reason_for_rejection:
+              <<: *application
```

Comment:
is this correct? seems odd for `reason_for_rejection` to have all the application attributes.